quickstarts/conviva/config.yml

```diff
@@ -0,0 +1,77 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: conviva
+
+# Displayed in the UI (required)
+title: NRI Conviva Integration
+
+# Long-form description of the quickstart (required)
+description: |
+  ## What is Conviva?
+  
+  [Conviva](https://www.conviva.com/) is a real-time analytics platform that
+  makes streaming data actionable
+
+  ## What is the NRI Conviva Integration?
+
+  The NRI Conviva integration is a New Relic Infrastructure
+  [on-host integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/get-started/introduction-host-integrations/)
+  that uses the [Conviva v3 Metrics API](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta)
+  to pull metrics from Conviva and push them into New Relic as dimensional
+  metrics.
+
+  The NRI Conviva integration supports the following Conviva v3 Metrics API
+  concepts.
+
+  * [Single metrics and metric groups](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#single-metrics-and-metric-groups)
+  * [Multiple singular metrics](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#retrieving-multiple-singular-metrics-in-a-request)
+  * [Dimensions](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#dimensions)
+  * [Time ranges](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#time-range)
+  * [Interval granularity](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#interval-granularity)
+  * [Filtering](https://developer.conviva.com/docs/metrics-api-v3/3e38d9ead39fc-metrics-v3-api-user-guide-beta#filtering-by-dimensions)
+
+  ## Getting Started
+
+  This quickstart is designed to help customers install and use the NRI Conviva
+  integration to collect and work with Conviva metrics.
+
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A New Relic quickstart designed to help customers install and use the NRI
+  Conviva integration to collect and work with Conviva metrics.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Scott DeWitt
+  - Chris McCarthy
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - conviva
+  - video
+  - streaming
+  - NRI
+  - integration
+
```

Comment:
The only one that really applies is `newrelic` since I work here.  We might want to consider having a "streaming" or "video" or "audio" or any of those since we integrate with a lot of streaming services.